recipes/pythia8/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pythia8" %}
+{% set version = "8.240" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://home.thep.lu.se/~torbjorn/{{ name }}/pythia{{ version|replace('.', '')  }}.tgz
+  sha256: d27495d8ca7707d846f8c026ab695123c7c78c7860f04e2c002e483080418d8d
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+  run:
+    - python
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - pythia8
+
+about:
+  home: http://home.thep.lu.se/Pythia/
+  license: GPL-2.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'A tool for the generation of events in high-energy collisions'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+      The Pythia program is a standard tool for the generation of events in
+      high-energy collisions, comprising a coherent set of physics models for
+      the evolution from a few-body hard process to a complex multiparticle
+      final state. It contains a library of hard processes, models for initial-
+      and final-state parton showers, matching and merging methods between hard
+      processes and parton showers, multiparton interactions, beam remnants,
+      string fragmentation and particle decays. It also has a set of utilities
+      and several interfaces to external programs.
+
+extra:
+  recipe-maintainers:
+    - henryiii
```

Yeah, I'm happy to be a maintainer.

```suggestion
    - henryiii
    - chrisburr
```